### PR TITLE
[Feat] Adds support for query in Fliplet.FormBuilder.getAll()

### PR DESCRIPTION
**Pull request by tonytlwu**
Adds similar support to https://github.com/Fliplet/fliplet-api/pull/4384 for using `Fliplet.FormBuilder.getAll(name)` with a query.

### CODE REVIEW

File: js/libs/form.js [hunk 1]
```diff
@@ -1013,6 +1013,14 @@ Fliplet.FormBuilder.get = function (name) {
   });
 };
 
-Fliplet.FormBuilder.getAll = function () {
-  return Promise.all(formBuilderInstances);
+Fliplet.FormBuilder.getAll = function (name) {
+  return Promise.all(formBuilderInstances).then(function(forms) {
+    if (typeof name === 'undefined') {
+      return forms
```
Comment (squallstar) [inline js/libs/form.js:1019]: missing semicolon

File: js/libs/form.js [hunk 2]
```diff
@@ -1013,6 +1013,14 @@ Fliplet.FormBuilder.get = function (name) {
...
+    }
+
+    return forms.filter(function(vueForm) {
```
Comment (squallstar) [inline js/libs/form.js:1022]: I don't think it's necessary to have the `vue` prefix here

Comment (tonytlwu) [reply js/libs/form.js:1022]: I don't think so either. I was following the previous pattern. I'll remove the prefix.

**Comment by tonytlwu** [comment]
@squallstar Ready for review

**Review by squallstar — APPROVED** [review]

**Pull request merged**